stdlib/inspect.pyi

```diff
@@ -117,18 +117,24 @@ if sys.version_info >= (3, 10):
 else:
     def signature(obj: Callable[..., Any], *, follow_wrapped: bool = ...) -> Signature: ...
 
+class _void: ...
+class _empty: ...
+
 class Signature:
-    def __init__(self, parameters: Sequence[Parameter] | None = ..., *, return_annotation: Any = ...) -> None: ...
-    # TODO: can we be more specific here?
-    empty: object
+    def __init__(
+        self, parameters: Sequence[Parameter] | None = ..., *, return_annotation: Any = ..., __validate_parameters__: bool = ...
+    ) -> None: ...
+    empty: ClassVar[Type[_empty]]
```

Comment:
Did you try just writing `empty = _empty`?

Reply:
yeah, I did and Python 3.10 complains
```shell
inspect.Signature.empty variable differs from runtime type Literal[<class 'inspect._empty'>]
```
and 3.8 and 3.9
```shell
inspect.Signature.empty variable differs from runtime type builtins.type
```

Reply:
I assume stubtest complains, but the other tests run? In that case, I'd suggest to just silence the stubtest warnings. But maybe @hauntsaninja has a different opinion?